README.md

```diff
@@ -56,11 +56,11 @@ new BundleAnalyzerPlugin(options?: object)
 |**`analyzerMode`**|One of: `server`, `static`, `disabled`|Default: `server`. In `server` mode analyzer will start HTTP server to show bundle report. In `static` mode single HTML file with bundle report will be generated. In `disabled` mode you can use this plugin to just generate Webpack Stats JSON file by setting `generateStatsFile` to `true`. |
 |**`analyzerHost`**|`{String}`|Default: `127.0.0.1`. Host that will be used in `server` mode to start HTTP server.|
 |**`analyzerPort`**|`{Number}`|Default: `8888`. Port that will be used in `server` mode to start HTTP server.|
-|**`reportFilename`**|`{String}`|Default: `report.html`. Path to bundle report file that will be generated in `static` mode. Relative to bundle output directory (which is `output.path` in webpack config).|
+|**`reportFilename`**|`{String}`|Default: `report.html`. Path to bundle report file that will be generated in `static` mode. Relative to bundle output directory (which is `output.path` in webpack config). Custom path example: `path/to/folder/report.html`. You can generate the `statsFileName` file outside of the bundle output directory by defining an absolute file path.|
```

Comment:
New description says `statsFileName`, but it should be `reportFilename`, right?

Reply:
Also, maybe we can just say something like `Path to bundle report file that will be generated in `static` mode. It can be either an absolute path or a path relative to bundle output directory (which is `output.path` in webpack config).`?